<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 61, 61);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QTextEdit:focus {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <property name="placeholderText">
    <string>Insira a Mensagem</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mensagem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_anexa">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>770</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QPushButton:hover {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <property name="text">
    <string>ANEXO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_enviar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>820</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QPushButton:hover {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <property name="text">
    <string>ENVIAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>610</y>
     <width>441</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_volta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QPushButton:hover {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <property name="text">
    <string>CADASTRO
 DE EMAIL</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>650</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	
	
	
	color: rgb(255, 255, 255);
	font: 13pt &quot;Arial&quot;;
	background-color: rgb(190, 190, 190);
	border-radius: 15px;
	text-align: center;


}

QProgressBar:chunk{ 
	background-color: rgb(255, 0, 127);
	border-radius :15px;


}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LIBERAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>850</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 127);</string>
   </property>
   <property name="placeholderText">
    <string>aula test</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_habilitar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>habilitar navegador</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>411</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>E-mail
Destinatario</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 87 14pt &quot;Arial Black&quot;;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QLineEdit:focus {
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_asunto">
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Emails Adicionais</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email_add">
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 87 14pt &quot;Arial Black&quot;;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QLineEdit:focus {
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email-Email</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_menos">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QPushButton:hover {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_mais">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 10pt &quot;Arial Black&quot;;
}
QPushButton:hover {
font: 87 10pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="styleSheet">
     <string notr="true">QTableWidget{color: rgb(255, 0, 127);
font: 87 10pt &quot;Arial Black&quot;;

}

QTableWidget:focus{
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_asunto_2">
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Assunto</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_asstuno">
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
font: 87 14pt &quot;Arial Black&quot;;
}
QLineEdit:focus {
font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Assunto</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWebEngineView" name="web_browser" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>561</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWebEngineView{
font: 87 14pt &quot;Arial Black&quot;;
border-radius: 18%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QWebEngineView:hover {
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 18%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressnavegar">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	
	
	
	color: rgb(255, 255, 255);
	font: 13pt &quot;Arial&quot;;
	background-color: rgb(190, 190, 190);
	border-radius: 15px;
	text-align: center;


}

QProgressBar:chunk{ 
	background-color: rgb(255, 0, 127);
	border-radius :15px;


}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>701</width>
     <height>33</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="botao_home">
    <property name="styleSheet">
     <string notr="true">QPushButton {
	qproperty-icon:url(D:/pytho willow/venda/icon/home-5-24.png);
	background-color: transparent;
	background-repeat: no-repeat;
	background-position: center;
	border: none;
}
QPushButton:hover {
	background-color: rgb(30,144,255);
}
QPushButton:pressed {
	background-color: rgb(1,84,149);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="voltar_site">
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	qproperty-icon:url(D:/pytho willow/venda/icon/arrow-left.png);
	background-color: transparent;
	background-repeat: no-repeat;
	background-position: center;
	border: none;
}
QPushButton:hover {
	background-color: rgb(30,144,255);
}
QPushButton:pressed {
	background-color: rgb(1,84,149);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="atualiza_site">
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	qproperty-icon:url(D:/pytho willow/venda/icon/reload.png);
	background-color: transparent;
	background-repeat: no-repeat;
	background-position: center;
	border: none;
}
QPushButton:hover {
	background-color: rgb(30,144,255);
}
QPushButton:pressed {
	background-color: rgb(1,84,149);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="avanca_site">
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	qproperty-icon:url(D:/pytho willow/venda/icon/arrow-right.png);
	background-color: transparent;
	background-repeat: no-repeat;
	background-position: center;
	border: none;
}
QPushButton:hover {
	background-color: rgb(30,144,255);
}
QPushButton:pressed {
	background-color: rgb(1,84,149);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_navega">
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 87 14pt &quot;Arial Black&quot;;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QLineEdit:focus {
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 6%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>NAVEGAR</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_envia_site">
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	qproperty-icon:url(D:/pytho willow/venda/icon/arrow-right.png);
	background-color: transparent;
	background-repeat: no-repeat;
	background-position: center;
	border: none;
}
QPushButton:hover {
	background-color: rgb(30,144,255);
}
QPushButton:pressed {
	background-color: rgb(1,84,149);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_caminho">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>770</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>caminho foto/arquivo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>770</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font: 87 12pt &quot;Arial Black&quot;;
border-radius: 2%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QLineEdit:focus {
	font: 87 14pt &quot;Arial Black&quot;;
color:white;
border-radius: 2%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>nome</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_formato">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>771</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	font: 87 10pt &quot;Arial Black&quot;;
border-radius: 2%;
border: solid 1px;
border-color:#b4b4b4;
border-width: 2px;
color:white;
}
QComboBox:focus {
	font: 87 8pt &quot;Arial Black&quot;;
color:white;
border-radius: 2%;
border: solid 1px;
border-color:#b4b4b4;
	background-color: rgb(180, 180, 180);
border-width: 2px;


}</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PNG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XLSX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JPG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TXT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PDF</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
